rasa/cli/data.py

```diff
@@ -234,6 +260,72 @@ def split_nlu_data(args: argparse.Namespace) -> None:
     telemetry.track_data_split(args.training_fraction, "nlu")
 
 
+def augment_nlu_data(args: argparse.Namespace) -> None:
+    """Load NLU training & evaluation data, paraphrases, an existing classification report, its corresponding config
+    file and suggest additional training
+     examples.
```

Comment:
First line of the docstring needs to be a short description followed by longer description in a separate paragraph.
https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#14d49d4fe8ec4b3796eb27a721417174